proposals/field-keyword.md

```diff
@@ -666,6 +656,10 @@ class B
 
 In the example above, binding to the backing field should result in an error: "initializer cannot reference non-static field".
 
+#### Answer
+
+`field` should not be a keyword in a property initializer. It will continue to bind (or fail to bind) as an identifier.
```

Comment:
```suggestion
`field` will not be a keyword in a property initializer. It will continue to bind (or fail to bind) as an identifier.
```

Reply:
Thanks! Copying in from Fred's notes now.